### DISCUSSION TITLE
change github education id

### DISCUSSION THREAD

**Discussion by sparshgoyal199** [community/community#142151] [GitHub Education]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

how can i change my id entered in github education

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @sparshgoyal199 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
